[Bug] 内部 lodash 依赖报错 Internal dependence lodash error

<!--
感谢您向我们反馈问题，为了高效的解决问题，我们期望你能提供以下信息：
-->

## What happens?

<!-- A clear and concise description of what the bug is. -->
<!-- 清晰的描述下遇到的问题。-->

安装最新的 `1.15.0` 后，内部报 lodash 的依赖错误 After installing the latest version `1.15.0`, got the dependency error of lodash
:

```
Module not found: Can't resolve 'lodash/isNull'
```

 报错截图如下 The screenshot of the error is as follows：
![image](https://github.com/ant-design/ant-design-web3/assets/13010589/31cd06e3-f2f1-4937-8d68-c1d237c6efe5)

在项目中另行按照 `lodash` 后错误消失 The error disappears after I install 'lodash' in my project;

<!-- 为节约大家的时间，无复现步骤的 ISSUE 会被关闭，提供之后再 REOPEN -->

## How To Reproduce

[复现例子 Reproduction](https://codesandbox.io/p/devbox/next-ant-web3-bug-reproduction-j8vv5d)

**Expected behavior** 1. 2.
不报错且不需要自行安装 `lodash` No errors and no need to install `lodash` in my project

<!-- 请提供复现链接/步骤，错误日志以及相关配置 -->

## Context

- **Ant Design Web3 Version**: 1.15.0
- **Node Version**: 20
- **Platform**: MacOS

--- a/.changeset/lovely-horses-obey.md
+++ b/.changeset/lovely-horses-obey.md
@@ -1,0 +1,5 @@
+---
+'@ant-design/web3': patch
+---
+
+fix: add lodash-es for @ant-design/web3

--- a/packages/web3/package.json
+++ b/packages/web3/package.json
@@ -52,7 +52,8 @@
     "bs58": "^5.0.0",
     "classnames": "^2.5.1",
     "copy-to-clipboard": "^3.3.3",
-    "decimal.js": "^10.4.3"
+    "decimal.js": "^10.4.3",
+    "lodash-es": "^4.17.21"
   },
   "devDependencies": {
     "@ant-design/web3-bitcoin": "workspace:*",
@@ -63,7 +64,8 @@
     "@types/react": "^18.2.78",
     "@types/react-dom": "^18.2.25",
     "father": "^4.4.4",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "@types/lodash-es": "^4.17.12"
   },
   "peerDependencies": {
     "antd": ">=5.0.0",

--- a/packages/web3/src/crypto-input/index.tsx
+++ b/packages/web3/src/crypto-input/index.tsx
@@ -2,7 +2,7 @@ import React, { useDeferredValue } from 'react';
 import type { Token } from '@ant-design/web3-common';
 import { theme as antdTheme, Flex, InputNumber, Typography } from 'antd';
 import Decimal from 'decimal.js';
-import { isNull } from 'lodash';
+import { isNull } from 'lodash-es';
 
 import { CryptoPrice } from '../crypto-price';
 import useIntl from '../hooks/useIntl';

